### ISSUE TITLE
MVI integration

### ISSUE THREAD

**Issue by ayaleloehr** [department-of-veterans-affairs/vets-api#244]
Using mock MVI service. Connect with @kreek on how to make this work.

**Comment by saneshark** [comment]
Blocked by: 

1) mock MVI service is only supported on dev
2) dev cannot handshake with MHV services

Unless dev firewall restrictions allow to communicate with MHV again or we can use the MockService on staging, we can't really test this one.

**Comment by ayaleloehr** [comment]
Dev firewall restrictions are not going to be opened unfortunately. We will only be using staging and prod for MHV, as Secure Messaging only has those 2 environments and we're not allowed to have 2 vets.gov environments both connect to 1 MHV environment apparently. Please let me know if you foresee large issues this will cause and I can try to push back more, but this is the answer I have gotten so far. 

With that, the mock MVI service will be going on staging until we get real connectivity there (hopefully next week).  @omgitsbillryan and/or @kreek are working on getting the mock MVI service onto staging. 

This issue is also being tracked in https://github.com/department-of-veterans-affairs/platform-team/issues/140

**Comment by ayaleloehr** [comment]
@saneshark Implemented for Rx [here](https://github.com/department-of-veterans-affairs/vets-api/pull/352), then reverted in [this](https://github.com/department-of-veterans-affairs/vets-api/pull/365). 

Still need to have it implemented for SM, although cannot merge that until step 4 is completed [here](https://github.com/department-of-veterans-affairs/kudos-team/issues/238).  Thinking maybe @mphprogrammer should do the SM implementation so he has visibility into this codebase and to limit @saneshark 's bus factor.

**Comment by aub** [comment]
@ayaleloehr I was planning on implementing that for SM once my PR [here](https://github.com/department-of-veterans-affairs/vets-api/pull/369) gets merged in. It should be a really simple duplication of the work @saneshark has already done for Rx.

**Comment by ayaleloehr** [comment]
@aub touch base quickly with @mphprogrammer first please. Should be fine but he might have started that so just want to be sure we don't duplicate the duplication

**Comment by robbiethegeek** [comment]
@ayaleloehr on the login is reenabled on RX we should be able to close.

**Comment by ayaleloehr** [comment]
https://github.com/department-of-veterans-affairs/vets-api/pull/417 got the 👍 so closing.
